docs/admin-manual/installation-startup-shutdown-reconfiguration.rst

```diff
@@ -191,7 +191,7 @@ This section is a brief description of DaemonCore. DaemonCore is a
 library that is shared among most of the HTCondor daemons which provides
 common functionality. Currently, the following daemons use DaemonCore:
 
--  *condor_master*
+-  :tool:`condor_master`
```

Comment:
Should this talk about the condor_master daemon create a reference to the tool? Likewise with the below condor_dagman